src/coreComponents/constitutive/contact/FrictionlessContact.hpp

```diff
@@ -118,7 +118,7 @@ class FrictionlessContact : public FrictionBase
    * @brief Create an update kernel wrapper.
    * @return the wrapper
    */
-  KernelWrapper createKernelWrapper() const;
+  KernelWrapper createKernelUpdates() const;
```

Comment:
do you need this change? I prefer `kernelWrapper`.

Reply:
The `InterfaceKernelBase` class inherits the `m_constitutiveUpdate` member from `KernelBase`, and it is initialized in the constructor as follows:

`KernelBase(SUBREGION_TYPE const & elementSubRegion,
           FE_TYPE const & finiteElementSpace,
           CONSTITUTIVE_TYPE & inputConstitutiveType)
  : m_elemsToNodes(elementSubRegion.nodeList().toViewConst()),
    m_elemGhostRank(elementSubRegion.ghostRank()),
    m_constitutiveUpdate(inputConstitutiveType.createKernelUpdates()),
    m_finiteElementSpace(finiteElementSpace)
{}`

All continuum constitutive models implement the `createKernelUpdates` method, unlike contact models, which use `createKernelWrapper`. I agree that `createKernelWrapper` might be clearer, however I believe that it’s important to maintain consistency between the methods used in solid and contact constitutive models.
Since modifying the `createKernelWrapper` method was less invasive, I opted for this solution.

Reply:
I see... WE should change them all to KernelWrapper IMO but we will do it another time.